tests/unit/dataactvalidator/test_b11_award_financial_3.py

```diff
@@ -12,19 +11,20 @@ def test_column_headers(database):
     assert (actual & expected_subset) == expected_subset
 
 
-def test_success(database):
-    """ Test valid object class code (3 digits) """
+def test_success_zero(database):
+    """ Test invalid object class code (3 digits) """
 
-    af = AwardFinancialFactory(object_class='object_class')
-    oc = ObjectClassFactory(object_class_code='object_class')
+    # This should return because if it's '000' a warning should be returned
+    af_1 = AwardFinancialFactory(object_class='0')
+    af_2 = AwardFinancialFactory(object_class='00')
 
-    assert number_of_errors(_FILE, database, models=[af, oc]) == 0
+    assert number_of_errors(_FILE, database, models=[af_1, af_2]) == 0
 
 
 def test_failure(database):
     """ Test invalid object class code (3 digits) """
 
     # This should return because if it's '000' a warning should be returned
-    af = AwardFinancialFactory(object_class='000')
+    af = AwardFinancialFactory(object_class='12345')
```

Comment:
We should still be getting a warning for "000" but we should also now get it for "0" and "00"